### PR TITLE
Fix up titles, descriptions

### PR DESCRIPTION
**Pull request by todb-r7**
This fixes up some of the titles and descriptions in recently landed modules. Of special note, the string 'towelroot' is mentioned in the title of the Android futex bug (since it seems like a pretty likely keyword for searching), and the version number was dropped from the Achat module. Note the Achat module seemed to be erroneously described as a "stack overflow," one of @wchen-r7's favorite pet peeves.
## Verification
- [x] See the new titles and descriptions make sense.
## Avoiding this in the future

As Batman says,

### CODE REVIEW

File: modules/exploits/android/local/futex_requeue.rb [hunk 1]
```diff
@@ -14,16 +14,17 @@ class Metasploit4 < Msf::Exploit::Local
...
   def initialize(info={})
     super( update_info( info, {
-        'Name'          => 'Android Futex Requeue Kernel Exploit',
+        'Name'          => "Android 'Towelroot' Futex Requeue Kernel Exploit",
         'Description'   => %q{
-            This module exploits a bug in futex_requeue in the linux kernel.
-            Any android phone with a kernel built before June 2014 should be vulnerable.
+            This module exploits a bug in futex_requeue in the Linux kernel, using
+            similiar techniques employed by the towelroot exploit. Any Android device
+            with a kernel built before June, 2014 is likely to be vulnerable.
```
Comment (wvu-r7) [inline modules/exploits/android/local/futex_requeue.rb]: ```June 2014```?

File: modules/exploits/windows/misc/achat_bof.rb [hunk 2]
```diff
@@ -13,13 +13,13 @@ class Metasploit3 < Msf::Exploit::Remote
 
   def initialize(info = {})
     super(update_info(info,
-      'Name'           => 'Achat v0.150 beta7 Buffer Overflow',
+      'Name'           => 'Achat Stack Based Buffer Overflow',
```
Comment (wvu-r7) [inline modules/exploits/windows/misc/achat_bof.rb]: ```Stack-Based```?

File: modules/exploits/windows/misc/achat_bof.rb [hunk 3]
```diff
@@ -13,13 +13,13 @@ class Metasploit3 < Msf::Exploit::Remote
...
       'Description'    => %q{
-        This module exploits an unicode SEH based stack buffer overflow in Achat v0.150. By
-        sending a crafted message to the default port 9256 it's possible to overwrites the
-        SEH handler. Even when the exploit is reliable it depends of timing since there are
+        This module exploits a Unicode SEH based stack based buffer overflow in Achat. By
```
Comment (wvu-r7) [inline modules/exploits/windows/misc/achat_bof.rb]: ```stack-based``` here, too.

File: modules/exploits/windows/misc/achat_bof.rb [hunk 4]
```diff
@@ -13,13 +13,13 @@ class Metasploit3 < Msf::Exploit::Remote
...
+        sending a crafted message to the default port UDP/9256, it's possible to overwrite the
```
Comment (wvu-r7) [inline modules/exploits/windows/misc/achat_bof.rb]: ```9256/udp``` is the traditional format.

File: modules/exploits/windows/misc/achat_bof.rb [hunk 5]
```diff
@@ -13,13 +13,13 @@ class Metasploit3 < Msf::Exploit::Remote
...
+        SEH handler. Even when the exploit is reliable, it depends of timing since there are
```
Comment (wvu-r7) [inline modules/exploits/windows/misc/achat_bof.rb]: ```depends on timing```.

File: modules/post/windows/gather/file_from_raw_ntfs.rb [hunk 6]
```diff
@@ -16,7 +16,8 @@ def initialize(info = {})
       'Name'         => 'Windows File Gather File from Raw NTFS',
       'Description'  => %q(
           This module gathers a file using the raw NTFS device, bypassing some Windows restrictions
-          such as open file with write lock. Can be used to retrieve files such as NTDS.dit.),
+          such as open file with write lock. Because it avoids the usual file locking issues, it can
+          be used to retrieve files such as NTDS.dit.),
```
Comment (wvu-r7) [inline modules/post/windows/gather/file_from_raw_ntfs.rb]: Maybe make it a brace and then drop it down a line to be more consistent and less confusing?

File: modules/exploits/android/local/futex_requeue.rb (hunk 1 above)
Comment (todb-r7) [inline modules/exploits/android/local/futex_requeue.rb]: [citation needed]

File: modules/exploits/windows/misc/achat_bof.rb (hunk 2 above)
Comment (todb-r7) [inline modules/exploits/windows/misc/achat_bof.rb]: $ grep -r "Stack Based" modules | wc -l
4
[ruby-2.1.5](master) 
todb@mazikeen:~/git/rapid7/metasploit-framework$ grep -r "Stack-Based" modules | wc -l
0
[ruby-2.1.5](master)

File: modules/exploits/android/local/futex_requeue.rb (hunk 1 above)
Comment (wvu-r7) [inline modules/exploits/android/local/futex_requeue.rb]: Here's one: http://www.ncsu.edu/ncsu/grammar/Comma3.html. :)

**Comment by todb-r7** [comment]
Updated.

File: modules/exploits/windows/misc/achat_bof.rb (hunk 2 above)
Comment (wvu-r7) [inline modules/exploits/windows/misc/achat_bof.rb]: FINE. YOU WIN.

File: modules/exploits/android/local/futex_requeue.rb (hunk 1 above)
Comment (todb-r7) [inline modules/exploits/android/local/futex_requeue.rb]: touche!

File: modules/post/windows/gather/file_from_raw_ntfs.rb [hunk 7]
```diff
@@ -14,9 +14,11 @@ class Metasploit3 < Msf::Post
   def initialize(info = {})
     super(update_info(info,
       'Name'         => 'Windows File Gather File from Raw NTFS',
-      'Description'  => %q(
+      'Description'  => %q{
           This module gathers a file using the raw NTFS device, bypassing some Windows restrictions
-          such as open file with write lock. Can be used to retrieve files such as NTDS.dit.),
+          such as open file with write lock. Because it avoids the usual file locking issues, it can
+          be used to retrieve files such as NTDS.dit.
+       },
```
Comment (wvu-r7) [inline modules/post/windows/gather/file_from_raw_ntfs.rb]: https://github.com/todb-r7/metasploit-framework/pull/14

**Comment by wvu-r7** [comment]
Done!

**Comment by todb-r7** [comment]
aw i fixed the whitespace thing, too, and force-pushed.

**Comment by wvu-r7** [comment]
Sorry for screwing up your branch. :X

**Pull request merged**